build.gradle

```diff
@@ -10,11 +10,13 @@
  */
 buildscript {
     ext {
-        opensearch_version = System.getProperty("opensearch_version", "1.0.0")
         distribution = 'oss-zip'
         opensearch_group = "org.opensearch"
-        opensearchVersion = '1.0.0'
-        common_utils_version = '1.0.0.0'
+        isSnapshot = "true" == System.getProperty("build.snapshot", "true")
+        opensearch_version = System.getProperty("opensearch_version", "1.1.0-SNAPSHOT")
```

Comment:
can we standardize this to match most other repos with opensearch.version instead?

Reply:
Ah yes, nice catch, I missed this one.

Reply:
https://github.com/opensearch-project/asynchronous-search/pull/32